backend/ctgov/script/etl.py

```diff
@@ -0,0 +1,161 @@
+import psycopg2
+import re
+
+from datetime import datetime
+from dotenv import load_dotenv
+from os import getenv
+from os.path import abspath
+from psycopg2 import Error
+
+dotenv_path = abspath("../../core/.env")
+print(dotenv_path)
+load_dotenv(dotenv_path)
+
+
+def print_current_time(type):
+    now = datetime.now()
+    dt_string = now.strftime("%H:%M:%S")
+    print(f"{type} time", dt_string)
+
+
+"""
+
+"""
+
+
+def connect_and_execute_psql(query, data):
+    try:
+        # Connect to an existing database
+        connection = psycopg2.connect(
+            database="aact",
+            user="postgres",
+            password=getenv("DB_PASSWORD"),
+            host="pgdb",
+            port=5432,
+        )
+
+        # This means there is a single query provided and the result set should return a single query
+        if isinstance(data, tuple) or isinstance(data, type(None)):
+
+            # Create a cursor to perform database operations
+            cursor = connection.cursor()
+            # Executing a SQL query
+            cursor.execute(query, data)
+
+            if query.__contains__("SELECT"):
+                # Fetch result
+                record = cursor.fetchall()
+                return record
+            else:
+                connection.commit()
+
+        # This means multiple queries were provided and that the data object will be more complex
+        elif isinstance(data, list):
+            cursor = connection.cursor()
+            for d in data:
+                cursor.execute(query, d)
+                connection.commit()
+
+    except (Exception, Error) as error:
+        print("Error while connecting to PostgreSQL", error)
```

Comment:
This doesn't really matter but for consistency
`print(f"Error while connecting to PostgreSQL {error}")`